test/lessify.js

```diff
@@ -53,7 +53,7 @@ test('should throw on invalid less', function(t) {
     , s = lessify('test.less');
 
   s.write('.}');
-  t.throws(function() { s.end(); }, new RegExp('missing opening `\\{`: test\\.less\\(1\\)'), 'should throw on invalid less');
+  t.throws(function() { s.end(); }, new RegExp('.*'), 'should throw on invalid less');
```

Comment:
good point. yeah, go ahead and remove the regex altogether. everything else looks good :)